server/main.py

```
@@ -0,0 +1,280 @@
+        return {
+            "filename": filename,
+            "metadata": metadata
+        }
+    
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error reading DTX file: {str(e)}")
+
+# Cloudflare Workers compatibility
+def on_fetch(request, env):
+    """Handler for Cloudflare Workers"""
+    import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)